ec2/InstanceType.dhall

```diff
@@ -304,6 +304,7 @@ let alternatives =
                 { permitLowerClassCPU : Bool
                 , permitWorseNetwork : Bool
                 , permitLosingLocalDisk : Bool
+                , permitCrossInstanceFamily : Bool
```

Comment:
you decided against adding `permitUpsizing` for now?

Reply:
Yes. I don't see the purpose in doing upsize when the `autoscaler` and `hpa` are looking at  the resources of the default family in use.